swap/peer.go

```diff
@@ -137,7 +131,6 @@ func (sp *Peer) processAndVerifyCheque(cheque *Cheque) (uint64, error) {
 
 	if err := sp.saveLastReceivedCheque(cheque); err != nil {
 		log.Error("error while saving last received cheque", "peer", sp.ID().String(), "err", err.Error())
-		// TODO: what do we do here?
```

Comment:
I think this is a super important one, please leave it there

Reply:
https://github.com/ethersphere/swarm/issues/1515

Reply:
@holisticode , I will put it back as you suggested.